.github/workflows/python-poetry-release.yaml

```
@@ -94,13 +94,15 @@ jobs:
           fetch-release-information: "true"
 
       - name: Commit and push pyproject.toml file
-        uses: bakdata/ci-templates/actions/commit-and-push@v1.0.0
+        uses: bakdata/ci-templates/actions/commit-and-push@v1.25.0
```

we removed the leading `v` in tags, remember? 😉 
```suggestion
        uses: bakdata/ci-templates/actions/commit-and-push@1.25.0
```

---

not for that version xD

---

oh right! 😄